external/netfx/netfx.depproj

```diff
@@ -7,21 +7,15 @@
     <BinPlaceRuntime>true</BinPlaceRuntime>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <NETStandardSupportPackageId>NETStandard.Library.NETFramework</NETStandardSupportPackageId>
-    <NETStandardSupportPackageVersion>2.0.0-preview2-25405-01</NETStandardSupportPackageVersion>
+    <NETStandardSupportPackageVersion>2.1.0-preview2-25630-02</NETStandardSupportPackageVersion>
```

Comment:
Is that right? Do we have a 2.1 version of this support package?

Reply:
Yes we do:

https://dotnet.myget.org/feed/dotnet-core/package/nuget/NETStandard.Library.NETFramework/2.1.0-preview2-25630-0

Reply:
Given we are moving the baseline up to 4.7.2 I think we can now remove the dependency on support package, correct @ericstj?

Reply:
> NETStandard.Library.NETFramework/2.1.0-preview2-25630-0

That build was one of the few builds from the 2.1 branch before it got turned off.  Don't use that version.  Use 2.0.1-servicing-26011-01, same as [CLI](https://github.com/dotnet/cli/blob/69062dda4edd7670cffa5c23876b38eccc130e09/build/DependencyVersions.props#L58) 

>  I think we can now remove the dependency on support package

We still need it for the TFMS listed below.

Reply:
>We still need it for the TFMS listed below.

I would only expect us to need it if we plan to run tests on those TFMs but we currently don't plan to as we are moving the baseline up to 472. I don't think we need it for compiling assets targeting those versions.